### PR TITLE
Feature/introjs impl

### PR DESCRIPTION
**Pull request by ronaldsg20**

### CODE REVIEW

File: src/components/pegout/PegOutForm.vue [hunk 1]
```diff
@@ -160,6 +164,97 @@ export default class PegOutForm extends Vue {
...
+      target: '#summary-estimated-fee',
+      content: `The estimated fee required by the protocol in ${this.environmentContext.getBtcTicker()}`,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#summary-btc-estimated-amount',
+      content: 'Based on the estimated fee and the amount transferred, this is the amount that will be in your destination address',
```
Comment (julia-zack) [inline src/components/pegout/PegOutForm.vue:242]: I'll change this text for: "..., this is the estimated final amount that will be transferred to the destination address."

File: src/components/pegout/PegOutForm.vue [hunk 2]
```diff
@@ -160,6 +164,97 @@ export default class PegOutForm extends Vue {
...
+      target: '#amount-field',
+      content: `Input the amount you want to convert into ${this.environmentContext.getBtcTicker()}`,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#max-btn',
+      content: 'If you want to send all your available balance in the wallet',
```
Comment (julia-zack) [inline src/components/pegout/PegOutForm.vue:186]: I'll change it for "If you want to send all the available balance in your wallet." or even "Click here if you want to send all the available balance in your wallet."
(i.e., the order of _your_ and _the_)

File: src/components/pegout/PegOutForm.vue [hunk 3]
```diff
@@ -160,6 +164,97 @@ export default class PegOutForm extends Vue {
...
+      target: '#summary-amount',
+      content: `This is the amount you will send to convert into ${this.environmentContext.getBtcTicker()}`,
+      params: {
+        highlight: true,
+        isLast: false,
+      },
+    },
+    {
+      target: '#summary-tx-fee',
+      content: `The estimated fee required by the network in ${this.environmentContext.getBtcTicker()}. Also called <strong>Gas</strong> `,
```
Comment (julia-zack) [inline src/components/pegout/PegOutForm.vue:218]: I'll put _gas_, with uncapitalized g.

**Comment by josedahlquist** [comment]
Can you schedule a demo of this intro feature please?

**Review by julia-zack — DISMISSED** [review]
LGTM

File: src/components/pegout/PegOutForm.vue [hunk 4]
```diff
@@ -160,6 +164,101 @@ export default class PegOutForm extends Vue {
 
   isReadyToSign = false;
 
+  tourSteps = [
+    {
+      target: '#wallet-connection',
+      content: `Click to select the rsk wallet where your ${this.environmentContext.getRbtcTicker()} are stored`,
```
Comment (lserra-iov) [inline src/components/pegout/PegOutForm.vue:170]: Maybe change rsk for RSK.

**Review by lserra-iov — DISMISSED** [review]

**Review by lserra-iov — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**